.github/workflows/_build.yml

```diff
@@ -74,30 +73,14 @@ jobs:
           pwd
           npm run build:staging
         working-directory: "./"
-      # build commonly used packages
-      - name: Build/compile common dependencies
-        if: ${{ inputs.treat-as-package == false }}
-        run: |
-          pwd
-          npm run build:cloud -w packages/shared
-          npm run build:cloud -w packages/commonwell-sdk
-          npm run build:cloud -w packages/api-sdk
-          npm run build:cloud -w packages/ihe-gateway-sdk
-          npm run build:cloud -w packages/carequality-sdk
-          npm run build:cloud -w packages/core
-        working-directory: "./"
```

Comment:
why is this no longer needed?

Reply:
We already [build from root before](https://github.com/metriport/metriport/pull/1522/files/b22354446b5b3a66bd8879b1c66b6c9d525e87c5#diff-ebfafa272202ca4d7cf36473df663e9f7a7cf844c549e60b2a6eae0a8be411edR63-R67), I think this is just duplicated and unnecessary work. Will validate when I merge it.